teekkari.py

```diff
@@ -314,6 +317,25 @@ def getPottiin(self, update: Update, context: CallbackContext):
             self.lastPottiin[userId] = now
             context.bot.sendMessage(chat_id=update.message.chat_id, text=msg)
 
+    def getTarot(self, update: Update, context: CallbackContext):
+        size = int(update.message.text.lower().split(' ')[1])
```

Comment:
This raises ValueError exception if the text isn't an integer, but bot luckily just handles that elsewhere. should this already have a try except block wrapping this, perhaps vittuilla takas if the parameter is not a number?